draft-ietf-moq-transport.md

```diff
@@ -481,6 +481,37 @@ MOQT scope, they can be used as a cache key for the track.
 If, at a given moment in time, two tracks within the same scope contain
 different data, they MUST have different names and/or namespaces.
 
+### Subscriptions
+
+There are two kinds of subscriptions in MoQT - a subscription to a track and a
+subscription to a track namespace.  Subscriptions to a track receive objects
+published on that track.  Subscriptions to a namespace recieve subscriptions to
+tracks within that namespace.
+
+A subscription can be initiated by either a publisher or a subscriber.  A
+publisher initiates a subscription to either a track or track namespace by
```

Comment:
A publisher can initiate a subscription to a track namespace?

Reply:
A publish initiated subscription to a track namespace = ANNOUNCE (namespace).